CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Issues
+- Please use the templates. If none of them is a perfect match, choose the closest one.
+
+# Security vulnerabilities
+- If you discover a security vulnerability, please do not file an issue. Instead, contact the Intranet developers directly. If you have no other means of doing this, send an email to intranet [at] tjhsst [dot] edu.
```

Comment:
Link to SECURITY.md instead of basically duplicating information. Some of this should be moved to that document as well.